src/components/__tests__/Modal.test.js

```diff
@@ -17,9 +17,7 @@ describe('Modal', () => {
     );
 
     expect(getByTestId('modal-backdrop').props.style).toEqual(
```

Comment:
Rather use custom matcher from `@testing-library/jest-native` for such cases.

```suggestion
    expect(getByTestId('modal-backdrop')).toHaveStyle({ backgroundColor: 'transparent' });
```